playbooks/show-package-drift.yml

```diff
@@ -23,11 +23,11 @@
         state: absent
       with_items: files.stdout_lines
       when: files.stdout_lines|count > 0
-      delegate_to: localhost 
+      delegate_to: localhost
       run_once: yes
 
     - name: get list of packages installed on host
-      shell: rpm -qa | sort -V
+      shell: rpm -qa | sort -V # ansible lint does not like this, wants to use yum module
```

Comment:
Can we change which lint rules are applied, and exclude this one? From ansible-lint's readme:
```
  -x SKIP_LIST          only check rules whose id/tags do not match these
                        values
```
Or perhaps we could go so far as to write our own list of rules (see: @eirslett's comment on #944)?
```
  -r RULESDIR           add one or more rules directories using one or more -r
                        arguments. Defaults to ['/Users/icordasc/sandbox
                        /ansible-lint/lib/ansiblelint/rules'] but any -r flags
                        completely override this.
```